test-resources/geo/location-schema.json

```diff
@@ -74,7 +76,7 @@
           "type": "integer"
         }
       },
-      "additionalProperties": true,
+      "additionalProperties": false,
```

Comment:
What's the reason for this change? I originally put this as `false` but it was amended after the discussion [here](https://github.com/ably/ably-asset-tracking-common/pull/1#discussion_r552633168).

Reply:
Good catch, sorry, I wasn't aware of that conversation :sweat_smile: 
I've changed it to `false` because after moving the `altitude` from the `properties` to the `coordinates` I've run tests and expected them to fail but they passed. I thought that's not good and decided to make the schema more strict but given that discussion you've mentioned, I'll change the `additionalProperties` back to `true` :wink: